prelude/__internal__/IO.mad

```diff
@@ -6,3 +6,25 @@ export trace = (v, a) => (#- { console.log(v); return a; } -#)
 
```

Comment:
Totes minor, but since you're touching this file anyways, `trace` should log both values:
```
export trace = (v, a) => (#- { console.log(v, a); return a; } -#)
```